CONTRIBUTING.md

```diff
@@ -1,17 +1,29 @@
-Contributing
-============
+# Contributing
 
-TODO: Fill out your contributing process, or just stick with the default.
-
-Steps
------
+## Steps
 
 Thanks for your interest in sensu-go cookbook! If you'd like to help out please:
 
 1. Fork it
 1. Create your feature branch (`git checkout -b my-new-feature`)
-1. Add tests for the new feature; ensure they pass (`rake`)
+1. Add tests for the new feature; ensure they pass See [TESTING.md](./TESTING.md)
 1. Add a changelog entry under `## [Unreleased]` and the appropriate sub header for the type of change per our [guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 1. Commit your changes (`git commit -am 'Add some feature'`)
 1. Push to the branch (`git push origin my-new-feature`)
 1. Create a new Pull Request
+
+## Maintainer
+
+Maintainers are GitHub Members who help review Pull Request and help organize and triage issues.
+
+### Pre Conditions
+
+1. Shown consistent and quality contributions to multiple repositories in the Sensu ecosystem.
```

Comment:
With the scope brought in a bit we should remove the word multiple repositories and make it specific to this repository, probably could be smith'd here a bit but maybe something along these lines:
```suggestion
1. Shown consistent and quality contributions to this repository as well as active engagement in the Sensu ecosystem.
```